### DISCUSSION TITLE
Suggestion: Mention adding local image to embed in docs

### DISCUSSION THREAD

**Discussion by Suven-p** [Rapptz/discord.py#5908] [General] [answered]
Would it be possible to include the info found here [FAQ - How do I use a local image file for an embed image?](https://discordpy.readthedocs.io/en/latest/faq.html#how-do-i-use-a-local-image-file-for-an-embed-image) in the embed section of docs [here](https://discordpy.readthedocs.io/en/latest/api.html#discord.Embed)? Or maybe point to the faq?

**Comment by Rapptz** [comment, marked as answer]
Idea seems fine to me. If you want to do this change, a pull request is the way to do it. It'd probably belong in the `set_image` documentation if anything, since it's locally bound to that method.
